Dockerfile.debian

```diff
@@ -0,0 +1,41 @@
+# before changing these variables, make sure the tag $PYTHON_VERSION-alpine$ALPINE_VERSION exists first
+# list of valid tags hese: https://hub.docker.com/_/python
+ARG PYTHON_VERSION=3.6
+ARG DEBIAN_VERSION=buster
```

Comment:
Why a Dockerfile for Debian?

Reply:
I think I didn't mean to commit this file. I was testing this on another branch, mainly for trying out other archs. I'll remove this file.

Reply:
Removed.